rules/library.bzl

```diff
@@ -308,7 +308,7 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
     weak_sdk_frameworks = kwargs.pop("weak_sdk_frameworks", [])
     sdk_includes = kwargs.pop("sdk_includes", [])
     pch = kwargs.pop("pch", "@build_bazel_rules_ios//rules/library:common.pch")
-    deps = [d for d in kwargs.pop("deps", [])]
+    deps = [] + kwargs.pop("deps", [])
```

Comment:
Got `Error: trying to mutate a frozen list value` if I use `deps = kwargs.pop("deps", [])` when building tests/framework/platforms/BUILD.bazel